docs/core/compatibility/cryptography/6.0/cipher-feedback-mode-persisted-key.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Breaking change: CreateEncryptor methods throw exception for incorrect feedback size"
+description: Learn about the breaking change in .NET 6 where the `CreateEncryptor` and `CreateDecryptor` methods for `AesCng` and `TripleDESCng` throw an exception for an incorrect feedback size when used with a persisted key for CFB mode.
+ms.date: 10/01/2021
+---
+# CreateEncryptor methods throw exception for incorrect feedback size
+
+The `CreateEncryptor` and `CreateDecryptor` methods for <xref:System.Security.Cryptography.AesCng> and <xref:System.Security.Cryptography.TripleDESCng> now throw a <xref:System.Security.Cryptography.CryptographicException> when the object instance is being used with a CNG persisted (named) key for Cipher Feedback (CFB) mode, with a feedback size other than eight (CFB8).
+
+## Previous behavior
+
+Previously, these classes allowed CFB128 (AesCng) or CFB64 (TripleDesCng) to be selected. However, if the key was a persisted key, then the computation was always done as if CFB8 was selected.
+
+## New behavior
+
+The `CreateEncryptor` and `CreateDecryptor` methods throw a <xref:System.Security.Cryptography.CryptographicException> when CFB128 or CFB64 mode is selected with these classes, and when backed by a persisted key.
```

Comment:
I feel like my interpretation of "when ..., and when" implies "additionally" (vernacular/disjoint 'and') not "when both of these are true" (logical 'and').

Maybe it needs to be flipped around?


... throw a CryptographicException when backed by a persisted key, the xref:SymmetricAlgorithm.Mode property is set to CipherMode.CFB and the xref:SymmetricAlgorithm.FeedbackSize property is set to a value other than 8.

Or maybe it's just my pre-coffee brain being weird.